src/Text/Pretty/Simple/Internal/OutputPrinter.hs

```diff
@@ -74,6 +74,16 @@ data OutputOptions = OutputOptions
   , outputOptionsEscapeNonPrintable :: Bool
   -- ^ Whether to replace non-printable characters with hexadecimal escape
   -- sequences.
+  , outputOptionsPrintStringLitsLiterally :: Bool
```

Comment:
Would it make sense to turn these two options into a data type with three constructors?

That might follow the flow of the code better below in the nested `if` statements.

Reply:
Good idea

Reply:
Done, feel free to suggest some better names.